spec/support/features/idv_helper.rb

```diff
@@ -69,9 +69,7 @@ def visit_idp_from_sp_with_ial2(sp, **extra)
   end
 
   def visit_idp_from_oidc_sp_with_ial2(
-    state: SecureRandom.hex,
-    client_id:,
-    nonce:,
+    client_id:, nonce:, state: SecureRandom.hex,
```

Comment:
I think probably easier to keep these all on separate lines, but keep the order change